tools/accuracy_checker/accuracy_checker/postprocessor/README.md

```diff
@@ -1,13 +1,15 @@
 # Postprocessors
 
-Postprocessor is function which processes prediction and/or annotation data after model infer and before metric calculation. For correct work postprocessors require specific representation format.
+Postprocessor is function which processes prediction and/or annotation data after model inference and before metric calculation. For correct work postprocessors require specific representation format.
```

Comment:
```suggestion
Postprocessor is class which processes prediction and/or annotation data after model inference and before metric calculation. For correct work postprocessors require specific representation format.
```

Reply:
possibly information about what the difference between adapter and postprocessor will be useful

Reply:
Done